EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -76,12 +76,12 @@ export default class EventsExpressService {
         return res;
     }
 
-    setContactUs=async(data)=>{
+    setContactUs = async (data) => {
         console.log('srv: ', data);
```

Comment:
Please remove all `console.log` statements, sine it is a bad practice to have such.